bitnami/mongodb-sharded/templates/config-server/config-server-statefulset.yaml

```diff
@@ -54,6 +54,9 @@ spec:
       {{- if .Values.configsvr.tolerations }}
       tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.configsvr.tolerations "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.configsvr.priorityClassName | quote }}
```

Comment:
The `quote` should not be in the `if` but in the next line. Check the others as well.

Reply:
:) rushed that one through - sorry about that.  fix commited